graph/src/blockchain/block_stream.rs

```diff
@@ -173,9 +173,22 @@ where
 }
 
 impl<C: Blockchain> BlockWithTriggers<C> {
-    pub fn new(block: C::Block, mut trigger_data: Vec<C::TriggerData>) -> Self {
+    pub fn new(block: C::Block, mut trigger_data: Vec<C::TriggerData>, logger: &Logger) -> Self {
         // This is where triggers get sorted.
         trigger_data.sort();
+
+        let old_len = trigger_data.len();
```

Comment:
https://docs.rs/itertools/latest/itertools/trait.Itertools.html#method.dedup_with_count

Reply:
I think this could be useful here

Reply:
Wow this is pretty cool! However I'll not use it because I only need to log the count previously and after, I think having to parse the result of `dedup_with_count` will be overkill.

Reply:
nvm you're right, it's not useful here